.github/workflows/presubmit-compositions.yaml

```diff
@@ -23,7 +23,42 @@ on:
     branches: ["master"]
     paths:
       - "experiments/compositions/**"
+
+env:
+  GOWORK: off
 jobs:
+  verify-goimports:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          stable: true
+      - uses: actions/checkout@v4
+      - name: verify goimports
+        run: |
+          ./experiments/compositions/scripts/github-actions/verify-goimports.sh
+  verify-gomod:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
+          stable: true
+      - uses: actions/checkout@v4
+      - run: |
+          ./experiments/compositions/scripts/github-actions/format-gomod.sh
+      - run: |
+          changes=$(git status --porcelain)
```

Comment:
Tip: I personally prefer to have the GitHub actions be 1:1 with scripts - I think if I was doing it today I would use `dev/ci/presubmits` and auto-generate the GitHub actions.  The advantage of doing so is that then it's easy to run the same thing as GitHub actions locally ... just run the script!

Reply:
happy to change it in a follow up.
Q: Is there an example i can look at. ?